### PR TITLE
add Application in the odo env view standard output

### PR DESCRIPTION
**Pull request by girishramnani**
**What type of PR is this?**
/kind bug

**What does this PR do / why we need it**:
see SUBJECT

**Which issue(s) this PR fixes**:

Fixes https://github.com/openshift/odo/issues/4574

**PR acceptance criteria**:

- [ ] Unit test 

- [X] Integration test 

- [ ] Documentation 

- [X] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Comment by girishramnani** [comment]
tests running since 11hrs
/retest

File: tests/integration/devfile/cmd_devfile_env_test.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ var _ = Describe("odo devfile env command tests", func() {
 
 	Context("When executing env view", func() {
 		It("Should view all default parameters", func() {
-			helper.CmdShouldPass("odo", "create", "nodejs", "--project", commonVar.Project)
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", commonVar.Project, "--app", "testing")
```
Comment (kadel) [inline tests/integration/devfile/cmd_devfile_env_test.go:31]: this is strange. the description is "should view all default parameters" but here you are setting `--app` explicitly.
Shouldn't this test verify that you can see the default application (app) in `odo env view` even if you don't set `--app`?

Comment (girishramnani) [reply tests/integration/devfile/cmd_devfile_env_test.go]: done, fixed

**Review by dharmit — APPROVED** [review]
/lgtm

**Pull request merged**